### PR TITLE
[Rule Tuning] Clearing or Disabling Windows Event Logs

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/392

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/defense_evasion_clearing_windows_event_logs.toml [hunk 1]
```diff
@@ -18,13 +18,14 @@ name = "Clearing Windows Event Logs"
...
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 type = "query"
 
 query = '''
 event.category:process and event.type:(start or process_started) and
-  process.name:wevtutil.exe and process.args:cl or
-  process.name:powershell.exe and process.args:Clear-EventLog
+  (((process.name:wevtutil.exe or process.pe.original_file_name:wevtutil.exe) and
+    process.args:(cl or clear-log or /e\:false)) or
+  process.name:powershell.exe and process.args:Clear-EventLog)
```
Comment (brokensound77) [inline rules/windows/defense_evasion_clearing_windows_event_logs.toml:28]:
```suggestion
  ((process.name:wevtutil.exe or process.pe.original_file_name:wevtutil.exe) and
    process.args:(cl or clear-log or /e\:false)) or
  (process.name:powershell.exe and process.args:Clear-EventLog)
```

**Review by brokensound77 — COMMENTED** [review]
can you also bump the `updated_date`

**Review by brokensound77 — APPROVED** [review]
Just the 2 comments then LGTM 👍

**Comment by Samirbous** [comment]
@brokensound77 not sure what caused this error ? 

image

File: rules/windows/defense_evasion_clearing_windows_event_logs.toml [hunk 2]
```diff
@@ -22,9 +22,9 @@ tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 type = "query"
 
 query = '''
-event.category:process and event.type:(start or process_started) and
-  process.name:wevtutil.exe and process.args:cl or
-  process.name:powershell.exe and process.args:Clear-EventLog
+event.category:process and event.type:(process_started or start) and 
+(process.name:wevtutil.exe or process.pe.original_file_name:wevtutil.exe) and 
+process.args:("/e:false" or cl or clear-log) or process.name:powershell.exe and process.args:Clear-EventLog
```
Comment (dstepanic17) [inline rules/windows/defense_evasion_clearing_windows_event_logs.toml:27]:
```suggestion
event.category:process and event.type:(process_started or start) and 
(process.name:"wevtutil.exe" or process.pe.original_file_name:"wevtutil.exe") and 
process.args:("/e:false" or cl or "clear-log") or process.name:"powershell.exe" and process.args:"Clear-EventLog"
```

Hey @Samirbous , I receive a Lucene syntax warning with the current KQL query.  It does look like the query is accurate, but we probably want to strive to not trigger any warnings for users that may cause confusion. It looks like some of the fields need quotes around them (keywords with special chars and process/file name. 

image

**Review by dstepanic17 — APPROVED** [review]
LGTM

**Pull request merged**